docs/getting_started.qmd

```diff
@@ -2,129 +2,21 @@
-So given a particular `Person` object, you first look up their household's MSOA
--- `msoa = population.households[ person.household ].msoa` and then look up
-flows for that MSOA -- `population.info_per_msoa[msoa].flows_per_activity`.
-
-Each person has exactly 1 flow for home -- it's just `person.household` with
-probability 1. A person has 0 or 1 flows to work, based on the value of
-`person.workplace`.
+SPC creates Google's [protocol buffer files](https://developers.google.com/protocol-buffers/docs/overview) a platform-neutral, extensible mechanism for serializing structured data in bi-directional way (forward-compatible and backward-compatible). SPC encodes the enriched synthetic population [schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto) in .proto files (.pb). You can read the "protobuf" (shorthand for a protocol buffer file) in any
+[supported language](https://developers.google.com/protocol-buffers/docs/overview#cross-lang),
+and then extract and transform just the parts of the data you want.
 
-This doesn't mean that all people in the same MSOA share the same travel
-behavior. Each person has their own `activity_durations` field, based on
-time-use survey data. Even if two people share the same set of places where
-they may go shopping, one person may spend much more time on that activity than
-another.
 
-See the [ASPICS conversion
-script](https://github.com/alan-turing-institute/uatk-aspics/blob/c8ad47cb01b084028b2f8ca90fcf078470efbee0/convert_snapshot.py#L166)
-for all of this in action -- it has a function to collapse a person's flows
-down into a single weighted list.
+## Areas ready to use and explore
 
-How do you interpret the probabilities/weights for flows? If your model needs
-people to visit specific places each day, you could randomly sample a venue
-from the flows, weighting them appropriately. For retail, you may want to
-repeat this sampling every day of the simulation, so they visit different
-venues. For primary and secondary school, it may be more appropriate to sample
-once and store that for the simulation -- a student probably doesn't switch
-schools daily.
+If all you need is to explore the enriched syntetic population file and mesuare if fits to you model, we have created the output file for **all counties in England**, so if the area you want to model is listed or you just want to explore the data, no need to run SPC yourself -- just [download data for your area](outputs.qmd).
 
-Alternatively, you can follow what ASPICS does. Every day, each person
-logically visits all possible venues, but their interaction there (possibly
-receiving or transmitting COVID) is weighted by the probability of each venue.
+If you are interested in create another area or combination of MSOAs, you will need then install and run SPC in your environment. We have tested SPC in Lunix and MacOS systems, for the moment there is no support for Windows systems, but we are working hard to address that.
```

Comment:
Nobody's working on making this run in Windows, and I don't think there are plans to unless somebody needs it. I think most Windows devs wind up using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) or something else. As long as they can figure out how to run commands in a terminal and get a basic dev environment with cmake set up, there shouldn't be problems.

Reply:
Ok.